docs/admin/authentication.md

```diff
@@ -294,19 +294,26 @@ To enable the plugin, configure the following flags on the API server:
 | `--oidc-issuer-url` | URL of the provider which allows the API server to discover public signing keys. Only URLs which use the `https://` scheme are accepted.  This is typically the provider's discovery URL without a path, for example "https://accounts.google.com" or "https://login.salesforce.com".  This URL should point to the level below .well-known/openid-configuration | If the discovery URL is `https://accounts.google.com/.well-known/openid-configuration`, the value should be `https://accounts.google.com` | Yes |
 | `--oidc-client-id` |  A client id that all tokens must be issued for. | kubernetes | Yes |
 | `--oidc-username-claim` | JWT claim to use as the user name. By default `sub`, which is expected to be a unique identifier of the end user. Admins can choose other claims, such as `email` or `name`, depending on their provider. However, claims other than `email` will be prefixed with the issuer URL to prevent naming clashes with other plugins. | sub | No |
+| `--oidc-username-prefix` | Prefix prepended to username claims to prevent clashes with existing names (such as `system:` users). Setting the value `oidc:` would create usernames like `oidc:jane.doe` | `oidc:` | No |
 | `--oidc-groups-claim` | JWT claim to use as the user's group. If the claim is present it must be an array of strings. | groups | No |
+| `--oidc-groups-prefix` | Prefix prepended to group claims to prevent clashes with existing names (such as `system:` groups). Setting the value `oidc:` would create group names like `oidc:engineering` and `oidc:infra`. | `oidc:` | No |
 | `--oidc-ca-file` | The path to the certificate for the CA that signed your identity provider's web certificate.  Defaults to the host's root CAs. | `/etc/kubernetes/ssl/kc-ca.pem` | No |
 
-If a claim other than `email` is chosen for `--oidc-username-claim`, the value
-will be prefixed with the `--oidc-issuer-url` to prevent clashes with existing
-Kubernetes names (such as the `system:` users). For example, if the provider
-URL is `https://accounts.google.com` and the username claim maps to `jane`, the
-plugin will authenticate the user as:
+If `--oidc-username-prefix` isn't provided and `--oidc-username-claim` is set to
```

Comment:
can we hoist this into the table? I expected to see the default behavior and opt out info by the param there

Reply:
yeah, update